src/components/resources/Patient/PatientSimple.js

```diff
@@ -1,49 +1,89 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import _get from 'lodash/get';
+import md5 from 'md5';
 
 import HumanName from '../../datatypes/HumanName';
-import { Root, Header, Body } from '../../ui';
-import { getId, getNames, getBirthDate, getGender } from './Patient';
+import Date from '../../datatypes/Date';
+import { Root, Header, Body, Value, Badge } from '../../ui';
+import './Patient.css';
 
-import './PatientSimple.css';
+function PatientContact(props) {
+  const { fhirData } = props;
+  const name = _get(fhirData, 'name');
+  const relationship = _get(fhirData, 'relationship[0].text');
 
-function PatientSimple(props) {
-  const { fhirResource } = props;
+  return (
+    <div>
+      <HumanName fhirData={name} />
+      {relationship && (
+        <small className="fhir-resource__Patient__patientContact-relationship">{` (${relationship})`}</small>
+      )}
+    </div>
+  );
+}
+
+export function getId(fhirResource) {
+  return _get(fhirResource, 'id');
+}
+export function getNames(fhirResource) {
+  return _get(fhirResource, 'name', []);
+}
+export function getBirthDate(fhirResource) {
+  return _get(fhirResource, 'birthDate');
+}
+export function getGender(fhirResource) {
+  return _get(fhirResource, 'gender');
+}
+
+function Patientsimple(props) {
```

Comment:
```suggestion
function PatientSimple(props) {
```
A minor thing - for now, we have a naming convention where we capitalize first letters of consecutive words. It would be good to stick to it if there isn't any problem.
(would need change in the whole file of course)